worker/task/process_retry_event_delivery_test.go

```diff
@@ -742,7 +757,7 @@ func TestProcessRetryEventDelivery(t *testing.T) {
 
 				l.EXPECT().CircuitBreaking().Times(1).Return(false)
 				l.EXPECT().AdvancedEndpointMgmt().Times(1).Return(true)
-				l.EXPECT().UseForwardProxy().Times(1).Return(false)
+				l.EXPECT().UseForwardProxy().Times(1).Return(true)
```

Comment:
Probably just lacking context but I don't understand why this expectation needed to change given that the test is still for at-least-once delivery.